pom.xml

```diff
@@ -41,9 +41,9 @@
   </licenses>
 
   <scm>
-    <connection>scm:git:ssh://github.com/jenkinsci/customtoosl-plugin.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/customtools-plugin.git</developerConnection>
-    <url>https://github.com/jenkinsci/customtools-plugin</url>
+    <connection>scm:git:git://github.com/jenkinsci/custom-tools-plugin.git</connection>
```

Comment:
https://help.github.com/articles/which-remote-url-should-i-use/#cloning-with-https-urls-recommended